Mage.Tests/src/test/java/org/mage/test/cards/copy/LazotepConvertTest.java

```diff
@@ -0,0 +1,46 @@
+package org.mage.test.cards.copy;
+
+import mage.constants.PhaseStep;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+import java.util.stream.Collectors;
+
+/**
+ * @author notgreat
+ */
+public class LazotepConvertTest extends CardTestPlayerBase {
+    @Test
+    public void testInvastionAmonkhetTransformed() {
+        addCard(Zone.BATTLEFIELD, playerA, "Badlands", 3);
+        addCard(Zone.BATTLEFIELD, playerA, "Underground Sea", 3);
+        addCard(Zone.GRAVEYARD, playerA, "Mutagen Connoisseur", 1);
+        addCard(Zone.HAND, playerA, "Invasion of Amonkhet");
+        addCard(Zone.HAND, playerA,"Char" );
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Invasion of Amonkhet");
+        waitStackResolved(1,PhaseStep.PRECOMBAT_MAIN);
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Char", "Invasion of Amonkhet");
+        //setChoice(playerA,true);
+        //addTarget(playerA,"Mutagen Connoisseur");
+        //These are auto-chosen by the AI
```

Comment:
Is there a reason the test won't work with `setStrictChooseMode(true)`?

Reply:
It was failing, yeah. It seemed like it had two copies of the replacement effect or something? I didn't investigate much since both manual testing and non-strict mode worked.

```
WARN  2023-07-06 21:53:32,019 Wrong choice command: Yes                                                                  =>[main] TestPlayer.assertWrongChoiceUsage 

Two Player Duel; T1.M1; active: PlayerA; stack: 1 (top: Cast Lazotep Convert using Transformed)
Unused choices:
Yes


java.lang.AssertionError: Missing CHOICE def for turn 1, step PRECOMBAT_MAIN, PlayerA
Lazotep Convert [32b]: you may have Lazotep Convert enter the battlefield as a copy of any creature card in a graveyard, except it's a 4/4 black Zombie in addition to its other types.
Lazotep Convert [32b]: you may have Lazotep Convert enter the battlefield as a copy of any creature card in a graveyard, except it's a 4/4 black Zombie in addition to its other types.
```

Reply:
It's a bug and must be fixed. Maybe bug is more complex (with copy or game engine), or maybe just with a card.